src/SFA.DAS.EAS.Web/Orchestrators/EmployerManageApprenticeshipsOrchestrator.cs

```diff
@@ -179,14 +180,20 @@ public sealed class EmployerManageApprenticeshipsOrchestrator : CommitmentsBaseO
             {
                 await AssertApprenticeshipStatus(accountId, apprenticeshipId);
 
-                var data = await _mediator.SendAsync(new GetApprenticeshipQueryRequest
+                var apprenticeshipData = await _mediator.SendAsync(new GetApprenticeshipQueryRequest
                 {
                     AccountId = accountId,
                     ApprenticeshipId = apprenticeshipId
                 });
 
-                AssertApprenticeshipIsEditable(data.Apprenticeship);
-                var apprenticeship = _apprenticeshipMapper.MapToApprenticeshipViewModel(data.Apprenticeship);
+                var commitmentData = await _mediator.SendAsync(new GetCommitmentQueryRequest
+                {
+                    AccountId = accountId,
+                    CommitmentId = apprenticeshipData.Apprenticeship.CommitmentId
+                });
+
+                AssertApprenticeshipIsEditable(apprenticeshipData.Apprenticeship);
```

Comment:
It's an existing method, but the name 'Assert..' is a bit loaded (it doesn't assert!).
Perhaps we could change it to something more meaningful, e.g. ThrowValidationExceptionIfNotEditable.